src/react/backend/location/LocationDetails/LocationDetailsAws.tsx

```diff
@@ -122,12 +122,13 @@ export default class LocationDetailsAws extends React.Component<
           <small>
             Store objects in the target bucket without a source-bucket prefix.
           </small>
-          <WarningInput hasError={!!this.state.bucketMatch}>
-            <small>
-              Storing multiple buckets in a location with this option enabled
-              can lead to data loss.
-            </small>
-          </WarningInput>
+          <WarningInput
+            error={
+              this.state.bucketMatch
+                ? 'Storing multiple buckets in a location with this option enabled\ncan lead to data loss.'
```

Comment:
nit: I feel using jsx here would be more elegant to avoid usage of `\n` within the text

Reply:
In the end, the newline wasn't taken into account, the CSS was ignoring the newlines. So I removed them everywhere